src/main/java/com/iexec/worker/feign/CustomFeignClient.java

```diff
@@ -86,7 +86,7 @@ public String getCoreVersion() {
     }
 
     //TODO: Make registerWorker return Worker
-    public void registerWorker(WorkerConfigurationModel model) {
+    public void registerWorker(WorkerModel model) {
```

Comment:
Maybe we could change this void method to a boolean one (registered true, already registered true)

Reply:
It will always be true, I'm not sure it is needed.

Reply:
Unlikely but up to you ..!